src/test/Makefile

```diff
@@ -17,9 +17,16 @@ ROOT = ../..
 OBJECT_DIR = ../../obj/test
 TARGET_DIR = $(USER_DIR)/target
 
+# Allow certain tests to be temporarily excluded, e.g. make TEST_EXCLUDED=atomic_unittest
+TEST_EXCLUDES := ?
+
 include $(ROOT)/make/system-id.mk
 include $(ROOT)/make/targets_list.mk
 
+ifeq ($(CYGWIN), 1)
+TEST_EXCLUDES := $(TEST_EXCLUDES) atomic_unittest
```

Comment:
The error that you are listing (`/usr/bin/ld: /cygdrive/d/My Documents/dev/embedded/betaflight/src/test/unit/atomic_unittest_c.c:23: undefined reference to `__imp__NSConcreteStackBlock'`) is not a bug in Betaflight (Betaflight does not define `__imp__NSConcreteStackBlock`, but a bug in cygwin. Have you filed a bug with cygwin so that this can get fixed and this workaround removed?